src/components/Navbar.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
+import { twMerge } from "tailwind-merge";
+import { FaInstagram, FaLinkedinIn } from "react-icons/fa";
+import { FaDiscord } from "react-icons/fa6";
+import { IoMenu } from "react-icons/io5";
+
+export default function Navbar() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const navLinks = [
+    {
+      label: "Events",
+      href: "/events",
+      isAuthRequired: false,
+    },
+    {
+      label: "Programs",
+      href: "/programs",
+      isAuthRequired: false,
+    },
+    {
+      label: "Opportunities",
+      href: "/jobs",
+      isAuthRequired: false,
+    },
+    {
+      label: "About Us",
+      href: "/about",
+      isAuthRequired: false,
+    },
+  ];
+
+  const socials = [
+    {
+      name: "Instagram",
+      href: "https://www.instagram.com/init.fiu/",
+      Icon: FaInstagram,
+      style:
+        "bg-gradient-to-tr from-orange-500 to-purple-500 hover:from-purple-600 hover:to-yellow-600",
+    },
+    {
+      name: "Discord",
+      href: "https://discord.gg/init",
+      Icon: FaDiscord,
+      style: "bg-primary-purple hover:bg-indigo-500",
+    },
+    {
+      name: "Linkedin",
+      href: "https://www.linkedin.com/company/initofficial/",
+      Icon: FaLinkedinIn,
+      style: "bg-blue-600 hover:bg-blue-700",
+    },
+  ];
+
+  return (
+    <nav className="grid p-8 max-w-screen-2xl mx-auto md:grid-cols-2 lg:grid-cols-3 ">
+      <div className="relative mb-7">
+        <Link href="/">
+          <Image
+            className="w-20"
+            src="/assets/images/logo.svg"
+            alt="Logo"
+            width={300}
+            height={120}
+            sizes="100vw"
+          />
+        </Link>
+
+        <button
+          className="absolute top-0 right-0 md:hidden"
+          onClick={() => setIsOpen(!isOpen)}
+        >
+          <IoMenu className="text-3xl" />
+        </button>
+      </div>
+
+      <div className={twMerge(!isOpen && "hidden", "mb-7 md:hidden lg:block")}>
```

Comment:
nav links disappear before menu icon is visible.
quick and dirty fix could be ...
 <div className={twMerge(!isOpen && "hidden", "mb-7  md:block")}>
        <ul className="flex flex-col relative md:-left-20  md:flex-row items-start 
			lg:items-center justify-evenly md:gap-2 ">